File: MIPPA/ClientApp/app/viewmodel/scorecard/playermatchviewmodel.ts
import { Player } from '../../model/player';

export class PlayerMatchViewModel {
    playerMatchId: number;
    homePlayer: Player;
    homePlayerScore: number;
    awayPlayer: Player;
    awayPlayerScore: number;
    homeBreaking: boolean;
    saved: boolean;
    validationError: boolean;
}

File: MIPPA/ViewModels/PlayerMatchViewModel.cs
namespace Mippa.ViewModels
{
    public class PlayerMatchViewModel
    {
        public int PlayerMatchId { get; set; }
        public PlayerViewModel HomePlayer { get; set; }
        public int HomePlayerScore { get; set; }
        public PlayerViewModel AwayPlayer { get; set; }
        public int AwayPlayerScore { get; set; }
        public bool HomeBreaking { get; set; }
        public bool Saved { get; set; }
    }
}